presentation-api/receiving-ua/support/PresentationReceiver_create_receiving-ua.html

```diff
@@ -0,0 +1,317 @@
+<!DOCTYPE html>
+
+<meta charset="utf-8">
+<title>Creating a receiving browsing context</title>
+<link rel="author" title="Tomoyuki Shimizu" href="https://github.com/tomoyukilabs/">
+<link rel="help" href="https://w3c.github.io/presentation-api/#creating-a-receiving-browsing-context">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../common.js"></script>
+<script src="stash.js"></script>
+<style>
+a { visibility: hidden; }
+</style>
+
+<iframe id="child" src="PresentationReceiver_create_receiving-ua_child.html"></iframe>
+<a href="PresentationReceiver_unreached_receiving-ua.html">do not navigate</a>
+<script>
+let finished = false;
+
+const sendResult = (tests, status, childLog) => {
+    if(!finished) {
+        finished = true;
+        const stash = new Stash(stashIds.toReceiver, stashIds.toController);
+        const log = childLog || document.getElementById('log').innerHTML;
+        stash.send(JSON.stringify({ tests: tests, status: status, log: log }));
+    }
+};
+
+add_completion_callback(sendResult);
+
+const child = document.getElementById('child');
+child.addEventListener('load', () => {
+    promise_test(t => {
+        t.add_cleanup(() => {
+            document.cookie = cookieName + '=False;Expires=' + new Date().toUTCString();
+            document.cookie = cookieNameChild + '=False;Expires=' + new Date().toUTCString();
+            sessionStorage.removeItem(storageName);
+            localStorage.removeItem(storageName);
+            sessionStorage.removeItem(storageNameChild);
+            localStorage.removeItem(storageNameChild);
+
+            if (db)
+                db.close();
+            indexedDB.deleteDatabase(dbName);
+            if ('serviceWorker' in navigator) {
+                navigator.serviceWorker.getRegistrations().then(registrations => {
+                    return Promise.all(registrations.map(reg => reg.unregister()));
+                });
+            }
+            if ('caches' in window) {
+                caches.keys().then(keys => {
+                    return Promise.all(keys.map(key => caches.delete(key)));
+                });
+            }
+        });
+
+        // Session history
+        assert_equals(window.history.length, 1, 'Session history consists of the current page only.');
+
+        // The Sandboxed auxiliary navigation browsing context flag
+        assert_equals(window.open('PresentationReceiver_unreached_receiving-ua.html'), null, 'Sandboxed auxiliary navigation browsing context flag is set.');
+
+        // The sandboxed top-level navigation browsing context flag (the following codes are expected to be ignored)
+        window.close();
+        document.querySelector('a').click();
+        location.href = 'PresentationReceiver_unreached_receiving-ua.html';
+
+        // The sandboxed modals flag (codes below are expected to be ignored)
```

Comment:
How is failure to ignore the following calls going to be detected?

I suspect the user agent would prompt the user if it does not ignore the calls and, provided the user does not answer the prompt, the test will fail with a timeout.

It could be good to be explicit about that:
1. by adding a comment as in "// If user agent prompts user, a timeout will occur and the test will eventually fail", and with user-friendly messages
2. by adding messages to users in the prompt as in "If you see this prompt, do not dismiss it and wait for test to time out"

Also `confirm` and `prompt` return a value (`false` and `null`), which could be checked as well.